e2e/features/submit-hearing-requirements.feature

```diff
@@ -246,3 +246,14 @@ Feature: Submit hearing requirements
 
     When I click the `Close and Return to case details` button
     Then I should see an alert confirming the case `has been updated with event: Submit hearing requirements`
+
```

Comment:
I think it is good to have the below scenario line to _submit-hearing-requirements-no-path_

Reply:
this scenario is handled now